### DISCUSSION TITLE
Won't load saved image

### DISCUSSION THREAD

**Discussion by AndyBlackburn** [tannerhelland/PhotoDemon#558] [Q&A]
PhotoDemon saved an edited image as a jpg file.
When I try to open it again for additional editing, PhotoDemon says I don't have permission.
When I try to click and drag the jpg image to PhotoDemon it says Please verify the images exist or use a supported form - but PD created the file!!
Can anyone tell me why is this happening?

**Comment by tannerhelland** [comment]
Hi @AndyBlackburn .  I'm sorry for the problem you've encountered!  I've never heard of an issue like this before.

Without more information, I'm not sure how much help I can be.  Here are some follow-up questions that can help.

- What was the exact error message you received?  
- What version of PhotoDemon are you using?  
- Can you open the file in any other software programs?  
- Is the file open in any other programs?  (Editing or viewing a file in multiple simultaneous programs can lead to unpredictable results.)

If an image file is locked by another program, PhotoDemon may not be able to access it.  This can sometimes happen temporarily after a file is saved, because a system app (like a virus scanner) locks down the file while it scans it.  The problem usually disappears after the virus scanner releases the file.

Permission problems can also happen if the file was saved to a folder with restricted access rights (an administrator or system location, for example).  

Any additional information you can provide would be helpful.  Thank you in advance.

**Comment by AndyBlackburn** [comment]
Hello Tanner,
1. Using "Open an image" - the error pops up in a box and says "Filename.jpg. You don't have permission to open this file. Contact the file owner or administrator to obtain permission."Using drag and drop - the error message says "Unfortunately, PhotoDemon was unable to load the following image(s) :Filename.jpg. Please verify that these image(s) exist, and that they use a supported image format (like JPEG or PNG). Thanks!"
2. PhotoDemon v.93. 
3. Yes, the files open in MS Paint, Paint.net, Gimp etc.4. 
4. No, the file is not open in other programs.
I get the same error messages when I try to open all my jpg, bmp, and tif files which are on different drives. The only image it will now load is a pcx file.
Very strange.
Since you say you are not aware of a problem like this, I'll simply uninstall the program, then install it again to see if the same reoccurs.Regards,Andy

    On Friday, 26 April 2024 at 17:24:54 BST, Tanner ***@***.***> wrote:  
 
 


Hi @AndyBlackburn . I'm sorry for the problem you've encountered! I've never heard of an issue like this before.

Without more information, I'm not sure how much help I can be. Here are some follow-up questions that can help.
   
   - What was the exact error message you received?
   - What version of PhotoDemon are you using?
   - Can you open the file in any other software programs?
   - Is the file open in any other programs? (Editing or viewing a file in multiple simultaneous programs can lead to unpredictable results.)

If an image file is locked by another program, PhotoDemon may not be able to access it. This can sometimes happen temporarily after a file is saved, because a system app (like a virus scanner) locks down the file while it scans it. The problem usually disappears after the virus scanner releases the file.

Permission problems can also happen if the file was saved to a folder with restricted access rights (an administrator or system location, for example).

Any additional information you can provide would be helpful. Thank you in advance.

**Comment by AndyBlackburn** [comment]
I have just completely removed PhotoDemon, and downloaded another copy of PhotoDemon v9
Unpacked and installed it, but again it will only load a pcx file - attempt other formats and it says I do not have permission.
Weird.
I'll see if earlier versions of PhotoDemon are available and try that.

**Comment by AndyBlackburn** [comment]
Version 8 the same
Even tried the Nightly Build - v 9.1 Alpha Build 380.
Will only load a pcx image, others I apparently do not have permission to access and/OR it asks to confirm they are supported file types.
My other graphics and photo editing software will load all of the ones PhotoDemon cannot.

**Comment by tannerhelland** [comment]
Thank you for the follow-up information, @AndyBlackburn .  The additional info is very helpful!

The permission error message does not come from PhotoDemon - that comes from your operating system.  

The image files you are trying to load are probably owned by an administrator account.  PhotoDemon is explicitly designed to *not* require administrator privilege levels to run.  This is a key part of what makes it a portable application.  

Your other photo editing applications are not portable applications, and were probably installed into the administrator's account directly.

I do not normally recommend this, but you could try right-clicking PhotoDemon and choosing "Run as administrator".  This might allow it to access the image files in question.

**Comment by AndyBlackburn** [comment]
Hello again Tanner,The access denial messages may indeed be generated by my system (Windows 10), but I have also used on-line image manipulation software as well as HD based programs without any denied access problems.I have only seen this problem recently whilst using PhotoDemon.
For your information.Andy


Sent from AOL on Android 
 
  On Tue, 30 Apr 2024 at 17:29, ***@***.***> wrote:   


Thank you for the follow-up information, @AndyBlackburn . The additional info is very helpful!

The permission error message does not come from PhotoDemon - that comes from your operating system.

The image files you are trying to load are probably owned by an administrator account. PhotoDemon is explicitly designed to not require administrator privilege levels to run. This is a key part of what makes it a portable application.

Your other photo editing applications are not portable applications, and were probably installed into the administrator's account directly.

I do not normally recommend this, but you could try right-clicking PhotoDemon and choosing "Run as administrator". This might allow it to access the image files in question.
